json/json.go

```diff
@@ -19,3 +24,20 @@ func Stringify(v interface{}, opts ...interface{}) string {
 	}
 	return string(buf)
 }
+
+// ReadFile Tries to open a file from the file system and read it into an object
+func ReadFile(f string, out interface{}) error {
+	if !fileutil.FileExists(f) {
+		return errors.New("File not found " + f)
+	}
+	js, err := os.Open(f)
+	defer js.Close()
+	if err != nil {
+		return err
+	}
+	bt, err := ioutil.ReadAll(js)
+	if err != nil {
+		return err
+	}
+	return json.Unmarshal(bt, &out)
```

Comment:
since `os.Open` returns a reader you could also do this `json.NewDecoder(f).Decode(&out)`.